csi/build.rs

```diff
@@ -8,5 +8,5 @@ fn main() {
         "src",                                    // output
         None,                                     // customizations
     )
-    .expect("Failed to compile gRPC definitions!");
+    .unwrap_or_else(|_| panic!("Failed to compile gRPC definitions!"));
```

Comment:
do not ignore the error
`|err| panic!("Failed to compile gRPC definitions, the error is: {}", e)`

Reply:
Done